.travis.yml

```diff
@@ -26,6 +26,7 @@ before_install:
 install:
   - composer self-update
   - pecl install -f mongodb-${DRIVER_VERSION}
+  - if [[ -z ${REQUIRE_SYMFONY_MESSENGER} ]]; then composer require --dev symfony/messenger --no-update; fi
```

Comment:
Could we use the same approach to installing `symfony/messenger` as DoctrineBundle? https://github.com/doctrine/DoctrineBundle/blob/a9e3720814d93789bd7c91e80957fcd1003bf3b6/.travis.yml#L52

Reply:
I tried it before but I had to copy all commands to each include item as the install section gets overridden. I would prefer my approach as it results in less duplicated lines.